### PR TITLE
whiptail: add page

### PR DESCRIPTION
**Pull request by sbrl**
I bring you `whiptail`, the default TUI dialog box command on most Debian-based systems. It should be installed by default, and you may have seen it when you update via `apt` in the terminal, asking you what you want to do if a configuration file got changed.

[source tutorial](http://xmodulo.com/create-dialog-boxes-interactive-shell-script.html)

### CODE REVIEW

File: pages/linux/whiptail.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+`{{result_variable_name}}="$(whiptail --title "{{title}}" --inputbox "{{message}}" {{height_in_chars}} {{width_in_chars}} {{default_text}} 3>&1 1>&2 2>&3)"`
+
+- Display a password input box:
+
+`{{result_variable_name}}="$(whiptail --title "{{title}}" --passwordbox "{{message}}" {{height_in_chars}} {{width_in_chars}} 3>&1 1>&2 2>&3)"`
+
+- Display a multiple-choice menu:
+
+`OPTION=$(whiptail --title "{{title}}" --menu "{{message}}" {{height_in_chars}} {{width_in_chars}} {{menu_display_height}} "{{value_1}}" "{{display_text_1}}" "{{value_n}}" "{{display_text_n}}" ..... 3>&1 1>&2 2>&3)`
```
Comment (pxgamer) [inline pages/linux/whiptail.md]: @sbrl, any reason this is `OPTION` instead of `{{result_variable_name}}` like the two examples above?

Comment (sbrl) [reply pages/linux/whiptail.md]: Yeah, because I forgot :P

I'll update it now.

Comment (pxgamer) [reply pages/linux/whiptail.md]: Oh 😋 Fair enough.

File: pages/linux/whiptail.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+`whiptail --title "{{title}}" --yesno "{{message}}" {{height_in_chars}} {{width_in_chars}}`
+
+- Customise the text on the yes / no buttons:
+
+`whiptail --title "{{title}}" --yes-button "{{text}}" --no-button "{{text}}" --yesno "{{message}}" {{height_in_chars}} {{width_in_chars}}`
+
+- Display a text input box:
+
+`{{result_variable_name}}="$(whiptail --title "{{title}}" --inputbox "{{message}}" {{height_in_chars}} {{width_in_chars}} {{default_text}} 3>&1 1>&2 2>&3)"`
```
Comment (mebeim) [inline pages/linux/whiptail.md]: Oh wow this is so useful and smart! I love it!

Comment (sbrl) [reply pages/linux/whiptail.md]: I don't acutally know precisely how all the shell redirections work, but it does lol

**Pull request merged**